sampledb.sql

```diff
@@ -98,7 +99,7 @@ PRIMARY KEY (`id`)
 
 CREATE TABLE `ensembl_release` (
 `release_id` INTEGER NOT NULL ,
-`version` INTEGER NOT NULL,
+`version` FLOAT NOT NULL,
```

Comment:
Currently I set this up to DECIMAL(10,1)

Reply:
When I change this type to `DECIMAL(10, 1)`, running the tests gives me this error:
```
SAWarning: Dialect sqlite+pysqlite does *not* support Decimal objects natively, and SQLAlchemy must convert from floating point - rounding errors and other issues may occur. Please consider storing Decimal numbers as strings or integers on this platform for lossless storage.
```
I think there will always be some differences between this test DB and the production one because this is sqlite and the production one is MySQL.  Could we just stick with `Float` here?

Reply:
We can, sure, imho, best course of action it to test against a mysql db. But will be another story.